oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/pipelined/PipelinedMongoDownloadTask.java

```diff
@@ -533,17 +581,33 @@ public boolean downloadInAscendingOrder() {
      * To download multiple ranges, create multiple instances of this class.
      */
     private class DownloadTask {
+        private final Stopwatch downloadTaskStart = Stopwatch.createUnstarted();
         private final DownloadOrder downloadOrder;
         private final DownloadStageStatistics downloadStatics;
+        private final long fromModified;
+        private final long toModified;
+        private final MongoParallelDownloadCoordinator parallelDownloadCoordinator;
         private long documentsDownloadedTotalBytes;
         private long documentsDownloadedTotal;
         private long totalEnqueueWaitTimeMillis;
```

Comment:
this is initialized to 0 and updated at line 830 but never used. Can we remove it?

Reply:
Done.